packages/app/src/runs/RunsContainer.vue

```diff
@@ -71,11 +95,29 @@ const props = defineProps<{
   gql: RunsContainerFragment
 }>()
 
+const isCloudProjectTypename = props.gql.currentProject?.cloudProject?.__typename === 'CloudProject'
```

Comment:
Same here - `props` should be reactive (like in React) so I don't think this variable is necessary. If your goal is just to make things concise, how about:

```ts
const isCloudProjectReturned = computed(() => props.gql.currentProject?.cloudProject?.__typename === 'CloudProject')
```
Then you don't need the second variable, `isCloudProjectReturned`, anymore.